database/migrations/2021_12_10_131813_clients.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class Clients extends Migration
+{
+/**
+* Run the migrations.
+*
+* @return void
+*/
+public function up()
+{
+DB::statement("
+CREATE TABLE `clients` (
+`id` BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+`first_name` VARCHAR (20) NOT NULL,
+`last_name` VARCHAR (20) NOT NULL,
+`membership_id` BIGINT UNSIGNED NULL,
```

Comment:
foreign keys is preferable to keep near primary key to improve readability.

Reply:
fixed